ProcessLib/ComponentTransport/CreateComponentTransportProcess.h

```diff
@@ -32,7 +37,8 @@ std::unique_ptr<Process> createComponentTransportProcess(
     BaseLib::ConfigTree const& config,
     std::vector<std::unique_ptr<MeshLib::Mesh>> const& meshes,
     std::string const& output_directory,
-    std::map<int, std::shared_ptr<MaterialPropertyLib::Medium>> const& media);
-
+    std::map<int, std::shared_ptr<MaterialPropertyLib::Medium>> const& media,
+    std::unique_ptr<ChemistryLib::ChemicalSolverInterface> const&
+        chemical_solver_interface);
```

Comment:
Clarification needed: With this declaration of the argument the `chemical_solver_interface` is an optional thing. Is it inteded?
If not, better to pass a `ChemistryLib::ChemicalSolverInterface const&`.